### PR TITLE
Rubocop friendly cops

### PR DESCRIPTION
**Pull request by DavidKang**
This PR is for solving some Rubocop complains. These are supposed to be not conflictive cops, which everyone most likely can agree on.

### CODE REVIEW

**Comment by bgeuken** [comment]
@openSUSE/open-build-service Please review and let us know, if there is any cop you don't agree on. 

Examples for the cops we apply here can be found here:
http://rubocop.readthedocs.io/en/latest/cops_layout/#layoutalignarray
http://rubocop.readthedocs.io/en/latest/cops_layout/#layoutalignhash
http://rubocop.readthedocs.io/en/latest/cops_layout/#layoutalignparameters
http://rubocop.readthedocs.io/en/latest/cops_layout/#layoutclosingparenthesisindentation
http://rubocop.readthedocs.io/en/latest/cops_layout/#layoutcommentindentation

http://rubocop.readthedocs.io/en/latest/cops_layout/#layoutemptylineaftermagiccomment
http://rubocop.readthedocs.io/en/latest/cops_layout/#layoutemptylineaftermagiccomment
http://rubocop.readthedocs.io/en/latest/cops_layout/#layoutemptylinesaroundbeginbody

File: src/api/app/views/webui/main/sitemap_projects.builder [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # rubocop:disable Metrics/LineLength
 xml.urlset(xmlns: "http://www.sitemaps.org/schemas/sitemap/0.9", "xmlns:xsi" => "http://www.w3.org/2001/XMLSchema-instance", "xsi:schemaLocation" => "http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd") do
-# rubocop:enable Metrics/LineLength
+  # rubocop:enable Metrics/LineLength
```
Comment (coolo) [inline src/api/app/views/webui/main/sitemap_projects.builder]: there is a cop about cop comments? :)

**Comment by bgeuken** [comment]
And keep in mind this is something everyone should agree on. So please wait with merging this.

File: src/api/app/views/webui/main/sitemap_projects.builder (hunk 1 above)
Comment (Ana06) [reply src/api/app/views/webui/main/sitemap_projects.builder]: about all comments 😉

**Comment by Ana06** [comment]
@bgeuken 

> And keep in mind this is something everyone should agree on. So please wait with merging this.

But not too much -> merge conflicts 😱

**Pull request merged**